suite-native/ethereum-tokens/src/utils.ts

```diff
@@ -1,3 +1,12 @@
 import { NetworkSymbol } from '@suite-common/wallet-config';
+import { ethereumTokenIcons, EthereumTokenIconName } from '@trezor/icons';
+
+import { EthereumTokenSymbol } from './types';
 
 export const isEthereumAccountSymbol = (symbol: NetworkSymbol) => symbol === 'eth';
+
+export const getEthereumTokenIconName = (symbol: EthereumTokenSymbol) =>
+    (symbol in ethereumTokenIcons ? symbol : 'erc20') as EthereumTokenIconName;
+
+export const convertTokenValueToDecimal = (value: string | number, decimals: number) =>
```

Comment:
Please move this function to @suite-native/formatters package, because it used only here and it shouldn't be used anywhere else expect formatters.

Also we should probaly use BigInt here

Reply:
I've moved the function to formatters package in 25fd663. But using `BigInt` is not possible since it can be used only for integer values and this function works with decimals. `BigNumber` would work, but we don't want to use this utility anymore because of performance, right?

Reply:
Sorry I meant `BigNumber`. We don't have any other choice unless we want to sacrifice precision which could lead to coin loss in some cases, performance comes as second in these cases.

Reply:
BigNumber applied in 0cb1a94f8